src/Worker.js

```diff
@@ -146,13 +146,21 @@ function run(data) {
             console.log(out); // eslint-disable-line no-console
           }
           if (!options.dry) {
-            fs.writeFile(file, out, function(err) {
+            const tmpFile = `${file}.tmp.${Math.floor(Math.random() * 1000000)}`;
```

Comment:
what do you think about using `os.tmpdir` with a stricter permission (0600) or even just the [`tmp`](https://www.npmjs.com/package/tmp) module's `file` method so we don't clutter the pwd in case of failure?

Currently if the write succeeds but the rename does not the randomly named transformed file will be left behind.